src/app/shared/store/app.actions.ts

```
@@ -1,5 +1,32 @@
+import { RegionAdmin } from 'shared/models/region-admin.model';
 import { TimerData } from 'shared/models/server-error';
+import { User } from 'shared/models/user.model';
+
+export class SetProfile {
+  static readonly type = '[app] set profile';
+  constructor(public payload: Parent | Provider | Employee | MinistryAdmin | RegionAdmin | AreaAdmin) {}
+}
+
+export class ClearProfile {
+  static readonly type = '[app] clear profile';
+  constructor() {}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)